ts/webui/src/static/interface.ts

```diff
@@ -203,6 +203,14 @@ interface Tensorboard {
     port: string;
 }
 
+// for TableList search
+interface SearchItems {
+    name: string;
+    operator: string;
+    value1: string;
```

Comment:
What are `value1` and `value2`

Reply:
value1 is to store firstInput val(user input), it maybe Trial id, Trial No. and parameter value
value2 is to strore secondInput val(user input), it is parameter seconde value, it's operator is `between`